### ISSUE TITLE
bulkDocs new_edits not working as documented

### ISSUE THREAD

**Issue by AgDude** [pouchdb/pouchdb#4759]
From my reading of the documentation, I expected the `new_edits` option to cause the existing document to be completely overwritten with the provided document. What appears to be happening instead if that the document is not updated at all (but doesn't raise a conflict error).

See this example:
http://jsbin.com/fuwamos/edit?html,js,console,output

**Comment by daleharvey** [comment]
hrm, I think you are right, will turn that into a test and see what CouchDB does with it

**Comment by daleharvey** [comment]
Actually given the test 

```
    it('4759 new_edits doesnt write new doc', function () {
      var db = new PouchDB(dbs.name);
      return db.put({_id: 'test', value: 1}).then(function(doc) {
        var newDoc = {_id: 'test', _rev: doc.rev, value: 2};
        return db.bulkDocs([newDoc], {new_edits: false});
      }).then(function() {
        return db.get('test');
      }).then(function(doc) {
        console.log(doc);
      });
    });
```
neither Couch or Pouch update the document, I feel like we have seen this reported before and it is expected behaviour, @nolanlawson do you remember?

**Comment by nolanlawson** [comment]
@daleharvey I believe that actually the CouchDB semantics are that, once you write `1-x`, you can never write `1-x` again (with that exact same revision number), and CouchDB will keep the original document. We have a test in place to confirm this behavior, but I can't recall where.

**Comment by nolanlawson** [comment]
Since this is one of those weird edge cases with CouchDB, where we'd prefer to stay consistent with Couch than to break new ground, I'd say this is a documentation bug.

**Comment by daleharvey** [comment]
Yup definitely want to keep consistency with couch here, opened a PR to update the docs - https://github.com/pouchdb/pouchdb/pull/4765

**Comment by nolanlawson** [comment]
fixed by 63decc3ab00748f8cb05572ec0084a65c565cd5c

**Comment by AgDude** [comment]
Thanks for looking into this and clarifying the docs.
